install-and-configure/install/kubecostv2.md

```diff
@@ -0,0 +1,51 @@
+# Kubecost 2.x Install/Upgrade
+
+Kubecost v2.0 introduced massive functionality changes including changes to backend architecture which may require additional changes to be made to your environment before upgrading an older version of Kubecost to 2.x. This article reviews several different common configurations and explains any necessary steps to take.
+
+## Single cluster users
+
+If you have a single cluster installation of Kubecost (i.e. one primary Kubecost instance on each cluster), then you can follow the standard upgrade process for Kubecost 2.x.
+
+If you are using Helm, it may look something like this:
+
+```sh
+```
+
+Upon upgrading, you should see the following pods running:
+
+```sh
+```
```

Comment:
I think something went wrong while copying and pasting content

Reply:
Still working on the doc! Haven't added these code blocks in yet

Reply:
Ok, just added them in!